src/components/Clock.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+type Props = {};
+
+interface State {
+  date: Date;
+}
+
+export class Clock extends React.Component<Props, State> {
+  timerId: NodeJS.Timer = setInterval(() => {}, 1000);
```

Comment:
Цей таймер буде працювати постійно, бо ми втратимо `id` до нього.
```suggestion
  timerId?: NodeJS.Timer;
```